Supervision Work/src/uk/ac/cam/ap886/oopjava/supervisionProjects/Hangman/Dictionary.java

```diff
@@ -0,0 +1,7 @@
+package uk.ac.cam.ap886.oopjava.supervisionProjects.Hangman;
+
+public interface Dictionary {
```

Comment:
Because it's really a hashtable that maps keys to values. Since java generics can't be void, in fact they need to be objects, this will be a huge waste of resources. Come to think of it I should've named it vocabulary.